Formula/ice.rb

```diff
@@ -12,6 +12,8 @@ class Ice < Formula
   end
 
   option "with-java", "Build Ice for Java and the IceGrid Admin app"
+  option "with-php", "Build Ice for PHP"
+  option "with-python", "Build Ice for Python"
```

Comment:
`depends_on :python => :optional if build.without?("python")` is definitely not a good way to do it. If you think it should be built with python by default, you should use `depends_on :python => :recommended` which will automatically produce a `--without-python` option.